azurerm/resource_arm_key_vault_access_policy_test.go

```diff
@@ -412,9 +412,7 @@ resource "azurerm_key_vault" "test" {
   resource_group_name = "${azurerm_resource_group.test.name}"
   tenant_id           = "${data.azurerm_client_config.current.tenant_id}"
 
-  sku {
-    name = "premium"
-  }
+  sku = "premium"
```

Comment:
shouldn't this be sku_name?
```suggestion
  sku_name = "premium"
```

Reply:
Yes... good catch...